woocommerce/Settings_API/Abstract_Settings.php

```diff
@@ -200,6 +200,28 @@ public function get_setting( $id ) {
 	}
 
 
+	/**
+	 * Updates the stored value for a setting.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param string $setting_id setting ID
+	 * @param array|bool|float|int|string $value
+	 * @throws Framework\SV_WC_Plugin_Exception
+	 */
+	public function update_value( $setting_id, $value ) {
+
+		$setting = $this->get_setting( $setting_id );
+
+		if ( ! $setting ) {
+			throw new Framework\SV_WC_Plugin_Exception( "Setting {$setting_id} does not exist" );
+		}
+
+		// performs the validations and updates the value
+		$setting->update_value( $value );
```

Comment:
Should we also call `$this->save( $setting_id )` here to update the option as well?

Reply:
I am not sure. I'll add a question to the story!

Reply:
Actually, since `delete_value` deletes the value from the DB, that makes sense. Thanks for pointing it out.